Jumpscale/clients/blockchain/tfchain/TFChainCapacity.py

```diff
@@ -128,10 +129,40 @@ def reserve_zos_vm(self, email, threebot_id, location, size=1, source=None, refu
             'type': 'vm',
             'location': location,
         })
+        _validate_reservation_vm(reservation)
+        return self._process_reservation(reservation, threebot_id, source=source, refund=refund)
+
+    def reserve_zdb_namespace(self, email, threebot_id, location, size=1,
```

Comment:
Misses a docstring.

Reply:
It is not clear to me what location is supposed to be.

Reply:
Same as the location for a 0-OS VM, can be a node id or a farm name.
I added doc sting so it's more clear